src/crh/bowe_hopwood/constraints.rs

```diff
@@ -27,7 +27,7 @@ pub struct ParametersVar<P: TEModelParameters, W: Window> {
     _window: PhantomData<W>,
 }
 
-pub struct CRHGadget<P: TEModelParameters, F: FieldVar<P::BaseField, ConstraintF<P>>>
+pub struct BoweHopwoodGadget<P: TEModelParameters, F: FieldVar<P::BaseField, ConstraintF<P>>>
```

Comment:
Why this renaming? I'd prefer if we didn't rename, as then the import becomes `bowe_hopwood::BoweHopwoodCRH`, which is repetitive. Right now, to use the CRH, we would use it as `bowe_hopwood::CRH`, which is slightly clearer in my opinion.

(This would also make it consistent with other things like `Parameters`, which doesn't get renamed.)

Reply:
fixed

Reply:
Hmm this is still called `BoweHopwoodGadget`, didn't we want to rename back to `CRHGadget`?

Reply:
my bad, I renamed poseidon's gadget name and accidentally marked this resolved...

I will fix it tomorrow

Reply:
should be fixed now. (I also refactored `pedersen` and `poseidon` similarly)

Reply:
(not sure if I need to refactor `injective_map` though. Now sure how to do)